test/integration_test.rb

```diff
@@ -44,6 +44,31 @@ def test_tests_are_selected_based_on_changes_in_current_working_tree
         output = rake('ttnt:test:run')
         assert_match '2 runs, 2 assertions, 2 failures', output[:stdout]
       end
+
+      def test_storage_file_resides_with_rakefile
+        tmpdir = 'tmp'
```

Comment:
That's right, thank you!
Fixed: https://github.com/Genki-S/ttnt/commit/769caff561008e27775e192fe64ecbc21462dcbe